membership_hr/__manifest__.py

```diff
@@ -10,13 +10,13 @@
     "website": "https://www.onestein.nl",
```

Comment:
Would be nice if this module is installed we have an applications menu in the membership menu, so people working with memberships don't have to switch to recruitement for it.

![image](https://github.com/user-attachments/assets/de45d4c3-6c1d-4f59-8479-652a4962c692)

Reply:
Added applications menu in members->configuration if membership_hr_recruitment module is installed